core/data/clients/database-client.go

```diff
@@ -27,7 +27,8 @@ import (
 
 type DatabaseType int8 // Database type enum
 const (
-	MONGO DatabaseType = iota
+	MONGO  DatabaseType = iota
+	INFLUX DatabaseType = iota
```

Comment:
You can drop the second `iota`, initialization will use implicit increment: https://github.com/golang/go/wiki/Iota